plugins/org.yakindu.sct.doc.user/src/user-guide/c-domain.textile

```diff
@@ -101,7 +97,8 @@ interface:
 On the right-hand side of the colon in the variable declaration, the variable's type must follow. In order to see which types are available, press @[Ctrl+Space]@. The content assist opens and shows the C types available in our C project, i.&#8239;e.
 * the C basic standard types,
 * the C99 types provided by including _stdint.h_,
-* the self-defined _Point_ type.
+* the self-defined _Point_ type,
+* the qualifiers _array_ and _pointer_.
```

Comment:
Attention: We recently decided that users __cannot__ declare arrays themselves, because we currently have no means of initializing them or knowing how big they are. They can only be imported from header files. Please fix this.